filesystem/filesystem.go

```diff
@@ -40,12 +39,11 @@ type FileServer struct {
 	Versioned bool
 	Files     map[protocol.FID]*File
 	IOunit    protocol.MaxSize
+
+	trace protocol.Tracer
```

Comment:
I guess this is ok. I kind of liked the idea of wrapping a server in a server, since it naturally recursed, and, further, a user could define their own 9p server and wrap the one they got in that. It's very plan 9-y to make servers wrap servers, where as this is more of a linux way of doing it. But if it works, it works.

Reply:
Do you mean having trace at multiple levels? It could be nice if the `debug.Server` somehow setup whatever it wrapped for tracing but I'm not sure how to pull that off.